### ISSUE TITLE
Syntax error in travelify/library/structure/content-extensions.php

### ISSUE THREAD

**Issue by punlf** [ColorlibHQ/travelify#1]
Dear Theme Author / Contributor,

I found a possible syntax error in travelify/library/structure/content-extensions.php 
In theme version 3.0.8, the following code in the previous version
`the_title( '', '', false )`
has been updated to
`the_title_attribute( '', '', false )`
For example, this code is present on lines 119 and 151.
However, the function the_title_attribute has a different syntax with the function the_title . 
To follow the syntax of the new function, the code should be like this:
`the_title_attribute( array('before' => '' , 'after' => '' , 'echo' => false ) )`

The syntax error has caused a problem as described in the wordpress support forum for the theme:
[Plain post title above post image (Error)](https://wordpress.org/support/topic/plain-post-title-above-post-image-error/)
In short, in archive page or category page, the title attribute of Image Thumbnail and "Read More" button does not work. And the title of the Thumbnail and "Read More" button, which should only appear on mouse hover, appears on top of the thumbnail and also before the "Read More" button.

Would you please check the code, and see if you can correct the code and update the theme.
Thank you very much.

**Comment by puikinsh** [comment]
Fixed in **3.1.0**, and your diagnosis was exactly right — including the line numbers.

`the_title_attribute()` was being called with `the_title()`'s argument list. Since it takes a single `$args`, the `''` meant "use the defaults", and the default for `echo` is `true`. So it printed the title to the page and returned nothing, which is why the `title` attribute came out empty *and* a stray title appeared above the thumbnail and before the Read More button.

All six call sites now pass `array( 'echo' => false )`:

https://github.com/ColorlibHQ/travelify/blob/v3.1.1/library/structure/content-extensions.php#L119
https://github.com/ColorlibHQ/travelify/blob/v3.1.1/library/structure/content-extensions.php#L151

(`before` and `after` already default to `''`, so they can be left out.)

Verified on a category and an archive page: the thumbnail link and the Read More button both carry a populated `title` attribute again, and there is no stray text in the meta bar. There's now a regression test asserting both.

Released in [v3.1.0](https://github.com/ColorlibHQ/travelify/releases/tag/v3.1.0), and [v3.1.1](https://github.com/ColorlibHQ/travelify/releases/tag/v3.1.1) is the current build.

Apologies this sat here for so long, and thank you for the report — it was accurate and complete, and it pointed straight at the cause.
